mne/epochs.py

```diff
@@ -841,9 +835,11 @@ def standard_error(self, picks=None):
 
         Parameters
         ----------
-        picks : array-like of int | None
-            If None only MEG, EEG, SEEG, ECoG, and fNIRS channels are kept
-            otherwise the channels indices in picks are kept.
+        picks : list | slice | None
+            Channels to include. If None, all **data** channels are used.
+            Slices and lists of integers will be interpreted as indices.
+            Valid channel type strings will pick good channels of the given
+            types. Valid channel name strings will pick the given channels.
```

Comment:
Here is the updated docstring

Reply:
@cbrnr you might also want to look since you expressed some interest in this feature

Reply:
Valid channel *type* strings -- i.e., 'meg' or 'eeg' -- will pick
good channels of the given types. Valid channel *name* strings
will pick the given channels.